client/client.lua

```diff
@@ -40,6 +43,24 @@ AddEventHandler("onResourceStart", function(resourceName) -- Used for when the r
 	end
 end)
 
+RegisterNetEvent('QBCore:Client:OnJobUpdate', function(job)
+    PlayerData.job = job
+end)
+
+RegisterNetEvent('ps-housing:client:liveRestart', function(properties)
+    for k, v in pairs(Config.Apartments) do
+        ApartmentsTable[k] = Apartment:new(v)
+    end
+
+    for k, v in pairs(properties) do
+        createProperty(v.propertyData)
+    end
+
+    TriggerEvent("ps-housing:client:initialisedProperties")
+    PlayerData = QBCore.Functions.GetPlayerData()
+    Debug("Initialised properties")
```

Comment:
@JnKTechstuff Majority of the code in this event is the same as whats being doing in InitialiseProperties. Can we just call that function here instead of duplicating the code?

Reply:
Looks like the only difference is this line `local properties = lib.callback.await('ps-housing:server:requestProperties')`, could just add a check if properties are passed in from arg use that else request